strata/cmd/mongo/lreplica_drivers/strata/main.go

```diff
@@ -6,13 +6,24 @@
 package main
 
 import (
+	"os"
 	"runtime"
+	"strings"
 
 	"github.com/facebookgo/rocks-strata/strata"
+	"github.com/facebookgo/rocks-strata/strata/cmd/mongo/lreplica_drivers/lrminiodriver"
 	"github.com/facebookgo/rocks-strata/strata/cmd/mongo/lreplica_drivers/lrs3driver"
 )
 
 func main() {
 	runtime.GOMAXPROCS(runtime.NumCPU())
-	strata.RunCLI(lrs3driver.DriverFactory{Ops: &lrs3driver.Options{}})
+
+	// Use a custom storage engine if set in the environment
+	// Use S3 as the default storage engine
+	switch strings.ToLower(os.Getenv("STORAGE_ENGINE")) {
```

Comment:
I put the storage setting in an environment variable because its credentials are being stored there too. Any time the remote storage changes, the creds will change too. So it'd be convenient to have these in the same place.

I see your point about the environment variable name. Should I change that to REMOTE_STORAGE and update the PR?